app/api/observations/observations.controller.js

```diff
@@ -98,3 +101,26 @@ exports.findObservation = route(async (req, res, next) => {
   req.observation = observation;
   next();
 });
+
+
+exports.findObservationUpdateDelete = route(async (req, res, next) => {
+  const where = {};
+  const user = req.user;
+  if (user.hasRole('volunteer')) {
+    where.user_id = user.id;
+  }
+
```

Comment:
could you make sure that if we are a owner we cannot delete any observation? It looks like we can retrieve any id if owner